dokumentation.tex

```diff
@@ -65,25 +66,31 @@
   Das Hintergrundbild für Charakterbögen ist nicht Bestandteil des Fanpakets.
   Standardmäßig erstellt die \LaTeX-Klasse Charakterbögen auf weißem Hintergrund
   (also auch ohne den Rand). Du kannst jedoch das Original-Hintergrundbild der
-  DSA-Charakterbögen verwenden; eine Anleitung findest du im
-  \texttt{README.html}. Hast die das Hintergrundbild in den Ordner 
-  \texttt{fanpaket} gelegt, wird es automatisch als Hintergrundbild für deine
+  DSA-Charakterbögen verwenden; Eine Anleitung findest du in der \texttt{README.html}.
```

Comment:
Die meisten Änderungen sind in Ordnung, aber nach dem Semikolon wird klein weitergeschrieben.